### PR TITLE
gui, lib/config: Add notification about new release channels

### PR DESCRIPTION
**Pull request by calmh**
### Purpose

As above

### Testing

Clicketyclick

### Screenshots

screen shot 2017-01-30 at 23 12 09

The link goes to https://build.syncthing.net/docs/223/users/releases.html

### CODE REVIEW

File: lib/config/config_test.go [hunk 1]
```diff
@@ -200,7 +200,7 @@ func TestOverriddenValues(t *testing.T) {
 		AlwaysLocalNets:         []string{},
 		OverwriteRemoteDevNames: true,
 		TempIndexMinBlocks:      100,
-		UnackedNotificationIDs:  []string{},
+		UnackedNotificationIDs:  []string{"channelNotification"}, // added in 17->18 migration
```
Comment (AudriusButkevicius) [inline lib/config/config_test.go]: Suggest you break this over multiple lines so that we end up with:

```
[]string{
  "foo", //bar
  "baz". //nuf
}
```

**Comment by AudriusButkevicius** [comment]
@st-review lgtm

**Comment by st-review** [comment]
@AudriusButkevicius: Noted! Need another LGTM or explicit merge command.

File: lib/config/testdata/v18.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<configuration version="17">
```
Comment (imsodin) [inline lib/config/testdata/v18.xml]: v18?

Comment (AudriusButkevicius) [inline lib/config/testdata/v18.xml]: Its 17 to 18 migration

Comment (imsodin) [inline lib/config/testdata/v18.xml]: Ok. I was simply comparing to v17.xml where the numbers matched, no "deeper insights" :)

Comment (calmh) [inline lib/config/testdata/v18.xml]: No you're right it should be 18. Fixed. Also another thing, the override test was flaky.

**Comment by calmh** [comment]
@st-review merge it

**Comment by st-review** [comment]
:ok_hand: Merged as 3eb7a9373ace20017b2bb4f781e94431f24089d9. Thanks, @calmh!

**Pull request closed without merging**